app/services/process_state.rb

```diff
@@ -0,0 +1,45 @@
+# ProcessState is an *experimental* thing that infers the state from
+# the state of application choices. Do not rely on this for business logic.
+class ProcessState
+  def initialize(application_form)
+    @application_form = application_form
+  end
+
+  def state
+    if states.empty?
+      :unsubmitted
```

Comment:
should we differentiate between this `unsubmitted`  and the following one?
- the first is maybe about not the application not yet completed
- the second is about awaiting the references

Reply:
No this is the same state. 

The second one is not about awaiting references, there's another state for that.